### DISCUSSION TITLE
Add option to make admonitions clickable

### DISCUSSION THREAD

**Discussion by grayswandyr** [pradyunsg/furo#724] [Ideas] [answered]
Hi
thanks for this nice theme. I'd like to be able to create "clickable" admonitions, that is admonitions that link to an external URL or an internal reference. Sphinx-design proposes "clickable cards" for this but the style of cards is completely different from that of Furo admonitions.

**Comment by pradyunsg** [comment]
Hmm... what's the use case for this? Could you provide a concrete example for why you'd want to do this?

**Comment by grayswandyr** [comment]
I'm co-authoring a book and we have beginner-oriented chapters and advanced chapters. In beginner chapters, we link to related advanced topics by using clickable cards that point to advanced chapters. Unfortunately, the style of clickable cards isn't consistent with furo. If admonitions were clickable, that would solve the issue, getting both clickability and a consistent theme.

**Comment by pradyunsg** [comment, marked as answer]
This is something that would be worked on upstream: https://github.com/sphinx-doc/sphinx/pull/12507

https://sphinx-design.readthedocs.io/en/latest/dropdowns.html exists today for similar functionality today though but yea... they are styled very differently,
